### PR TITLE
fix #76598 webbrowsertools.com

### LINKED ISSUE #76598: webbrowsertools.com
### Issue URL (Ads)
 [https://webbrowsertools.com/popup-blocker/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwebbrowsertools.com%2Fpopup-blocker%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish, <br/>AdGuard French, <br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Dutch, <br/>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by wout-codes**
#76598

### CODE REVIEW

File: EnglishFilter/sections/css_extended.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 !
 ! Extended CSS rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/76598
+webbrowsertools.com#?#div[class*="card shadow"]:contains(Advertisement)
```
Comment (Alex-302) [inline EnglishFilter/sections/css_extended.txt:5]: `*` is not requied. `^` is enough

Comment (wout-codes) [reply EnglishFilter/sections/css_extended.txt:5]: Sure.

Comment (wout-codes) [reply EnglishFilter/sections/css_extended.txt]: @Alex-302 Btw, do they even update the Popup Blocker Userscript?

**Review by Alex-302 — APPROVED** [review]

File: EnglishFilter/sections/css_extended.txt (hunk 1 above)
Comment (Alex-302) [reply EnglishFilter/sections/css_extended.txt]: @wout-codes 
> Btw, do they even update the Popup Blocker Userscript?

What do you mean?

Comment (wout-codes) [reply EnglishFilter/sections/css_extended.txt]: @Alex-302 The AdGuard's Popup Blocker UserScript.

Comment (Alex-302) [reply EnglishFilter/sections/css_extended.txt]: About popup blocker better write to https://github.com/AdguardTeam/PopupBlocker
too many notifications in this repository, I can miss reply

**Pull request merged**